src/html/HtmlReporter.js

```diff
@@ -112,12 +112,16 @@ jasmineRequire.HtmlReporter = function(j$) {
     this.specDone = function(result) {
       stateBuilder.specDone(result);
 
-      if (
-        noExpectations(result) &&
-        typeof console !== 'undefined' &&
-        typeof console.error !== 'undefined'
-      ) {
-        console.warn("Spec '" + result.fullName + "' has no expectations.");
+      if (noExpectations(result) && typeof console !== 'undefined') {
```

Comment:
I think we can probably do away with the checks for `console` and its methods. Jasmine no longer supports any environments that don't provide them.